src/components/SectionHeader/SectionHeader.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./SectionHeader.scss";
```

Comment:
All style imports should be placed after all module ones. Move it to the bottom

Reply:
Done